CDCgov/prime-reportstream
# Adding handling for monkeypox AOE ages

This PR adds internal fields for patient_age_calculated (from date of birth) and patient_age_aoe (for OBX-3 LOINC code 30525-0) and populates patient_age from the AOE if it is present, else it populates age calculated from the date of birth.

Test Steps:
1. Test with production Aegis (DOB only), Mayo (19000101 DOB + age AOE), and Sonic (valid DOB + age AOE). Confirm Aegis and Sonic output to DCIPHER are unchanged before/after. Mayo changes from 122 to the actual age. 

**Note, however, that Mayo is currently sending the AOE OBX under the specimen repetition, not the observation repetition. We need to have them change this before it will work. They can send the AOE OBX immediately after the result OBX or after the NTE segments, but they can't send after the SPM.**


## Changes
- Adds patient_age_aoe and patient_age_calculated and to monkeypox.schema. Adds age to monkeypox.schema, populated with either patient_age_aoe and patient_age_calculated.
- Removes the patient_age mapper logic from dcipher-monkeypox.schema, since it now lives in monkeypox.schema
- Updates the generated documentation files

## Checklist

### Testing

## Linked Issues
- Fixes #6732
